docs/user-guide/index.md

```diff
@@ -0,0 +1,3 @@
+# User Guide
+
+- [Ensure content best practices](/user-guide/content_best_practices.md)
```

Comment:
This won't link correctly from here

```suggestion
- [Ensure content best practices](content_best_practices.md)
```

Reply:
@Qalthos In fact if using relative mkdocs will produce a warning that is translated to a an error due to use of `--strict`, try it yourself!